### DISCUSSION TITLE
How to enable OT shell and updatehub shell on USB CDC ACM?

### DISCUSSION THREAD

**Discussion by nehalrohit** [zephyrproject-rtos/zephyr#30689] [Q&A] [answered]
Hello
Thanks for all your support,
I want to USB console as the same as the UART console for the openthread example.
I am able to print all log on USB CDC but not cmd shell on USB.
![img_20201214](https://user-images.githubusercontent.com/50902801/102070856-b6695300-3e25-11eb-8a62-aac5fa8054f5.png)

How to enable OT shell and updatehub shell on  USB?

**Comment by jfischer-no** [comment, marked as answer]
`CONFIG_UART_SHELL_ON_DEV_NAME="CDC_ACM_0"`
You can use _tests/bluetooth/shell/cdc_acm.conf_ as example.
